lib/SPIRV/SPIRVToLLVMDbgTran.cpp

```diff
@@ -49,6 +49,7 @@
 #include "llvm/IR/DIBuilder.h"
 #include "llvm/IR/IntrinsicInst.h"
 #include "llvm/IR/Module.h"
+#include <bit>
```

Comment:
on main branch this include was removed, see:
commit d194b9d4e60bc33c10e0ed999d9e3b1b6daaf857
Author: Jinsong Ji <jinsong.ji@intel.com>
Date:   Wed Jul 3 11:32:27 2024 -0700

    Remove unecessary include that require c++20 (#2628)

do we still need it here?

Reply:
Thanks for catching this. I thought https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/2647/commits/63deb0de8f593c2ff67736c15c3077a48a46ea78 was enough for not requiring c++20.